.github/workflows/maven.yml

```diff
@@ -286,6 +300,11 @@ jobs:
           if [[ ${{matrix.benchmark}} == templated ]]; then
             java -jar benchbase.jar -b tpcc -c config/mysql/sample_tpcc_config.xml --create=true --load=true --execute=false --json-histograms results/histograms.json
             java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/mysql/sample_${{matrix.benchmark}}_config.xml --create=false --load=false --execute=true --json-histograms results/histograms.json
+          # For anonymization, we load tpcc and anonymize a single table. The workload itself is not executed
+          elif [[ ${{matrix.benchmark}} == anonymization ]]; then
+            java -jar benchbase.jar -b tpcc -c config/mysql/sample_tpcc_config.xml --create=true --load=true --execute=false --json-histograms results/histograms.json
+            java -jar benchbase.jar -b tpcc -c config/mysql/sample_${{matrix.benchmark}}_config.xml --anonymize=true
```

Comment:
Why aren't we executing the workload?  Is that a future PR?

Reply:
With only the skeleton, the anonymization does not do anything. Once fully implemented, it would create an anonymized **copy** of one table. The original table would then have to be replaced such that running the benchmark would make sense. For now I would suggest to leave it out

Reply:
Please add comments to that effect.  We can remove them in the future PR when you enable the rest of the framework and tests.